src/main/java/uk/gov/justice/probation/courtcasematcher/model/domain/CourtCase.java

```diff
@@ -37,8 +39,6 @@
 
     private final String probationStatus;
 
-    private final String probationStatusActual;
```

Comment:
This is what was used to save, as opposed to display. Is this part of a refactor so that we can stop mixing our JPA entity and JSON (output) objects ?

Reply:
Yeah, my reasoning here is that as this is the domain model it's not really interested in the display status as it doesn't need to display it anywhere. In practice it *is* probationStatusActual that it's interested in but I didn't want to name it that as that's a slightly shonky implementation detail of the court-case-service which ideally we wouldn't allow to bleed into the matcher domain model.